aea/mail/base.py

```diff
@@ -762,6 +762,25 @@ def get(
         except queue.Empty:
             raise Empty
 
+    async def async_get(self) -> Envelope:
+        """
+        Get an envelope async way.
+
+        :return: the envelope
+        """
+        try:
+            return await self.in_queue.async_get()
+        except queue.Empty:
+            raise Empty
```

Comment:
Are we sure we want to label all coroutines with `async` in the method name?

Reply:
@DavidMinarsch it's about to discuss, cause get,put already present.